bellbot_gui/www/feedback.html

```diff
@@ -4,8 +4,8 @@
 <link rel="stylesheet" type="text/css" href="css/style.css">
 <script src="js/jquery-1.11.2.min.js"></script>
 
-<script src="roslibjs/include/EventEmitter2/eventemitter2.js"></script>
-<script src="roslibjs/build/roslib.js"></script>
+<script src="http://cdn.robotwebtools.org/EventEmitter2/current/eventemitter2.min.js"></script>
+<script src="http://cdn.robotwebtools.org/roslibjs/current/roslib.min.js"></script>
 <script src="ros2djs/include/EaselJS/easeljs.js"></script>
 <script src="ros2djs/build/ros2d.js"></script>
```

Comment:
As far as I can see you only uses services and topics in this file. If so, you don't need ros2djs/include/EaselJS/easeljs.js or ros2djs/build/ros2d.js if I'm not mistaken. This will also let the page load quicker.